content/post/2024-11-06-again.md

```
@@ -0,0 +1,48 @@
+3. [American War](https://amzn.to/3NUHwPZ) by Omar El Akkad
+4. [Nuclear War: A Scenario](https://amzn.to/3UAKT1Z) by Annie Jacobsen
+
+## Ugh
+
+Yep.
+
+## Now what?
+
+No idea. Here is a good [post](https://www.popehat.com/p/and-yet-it-moves) that helped me think through where we may be going. It isn't pretty.
+
+![](/images/posts/worse.jpg)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)